### ISSUE TITLE
WhatsApp group message: only one receiver

### ISSUE THREAD

**Issue by ffwraspi** [tgalal/yowsup#1715]
hello together,

i want to send a group message in whatsapp with yowsup.

the group includes 3 members.
the raspberry/yowsup client and two normal whatsapp users (mobile phones)

this is my code in the python script

yowsup-cli demos -c config -s 49XXXXXXXXXX-XXXXXXXXXXX@g.us "Hello"

if i send the message from the raspberry/yowsup client, only one mobile phone receiver is receiving the message.

The second mobile phone client receives nothing =(

Maybe somebody can help me.

thanks a lot in advance

**Comment by sowerkoku** [comment]
you tested only with:
`yowsup-cli demos -c config -s 49XXXXXXXXXX-XXXXXXXXXXX "Hello"`

**Comment by ffwraspi** [comment]
Hi sowerkoku,

i testet it with

yowsup-cli demos -c config -s 49XXXXXXXXXX-XXXXXXXXXXX "Hello"
and
yowsup-cli demos -c config -s 49XXXXXXXXXX-XXXXXXXXXXX@g.us "Hello"

both have the same result....

**Comment by sowerkoku** [comment]
now well understood.
the message reaches the group, but only see the message number.
send test messages between normal numbers.
I think the problem is the member of the group.

**Comment by ffwraspi** [comment]
Hi,

thanks for your answer.
I have tried your suggestion.
I have send messages between all 3 clients.
After that I tried to send a group message from the raspberry pi.
Unfortunately only one mobile phone recives the group message =(

**Comment by sowerkoku** [comment]
try with cli:
yowsup-cli demos -c config -y

**Comment by schoerg** [comment]
Update yowsup

**Comment by sowerkoku** [comment]
I assumed that you have the latest version.

**Comment by jlguardi** [comment]
The best way to identify the issue is add a new participant to the group using yowsup. So you will receive the actual message send to participants.

**Comment by jlguardi** [comment]
The best way to identify the issue is add a new participant to the group using yowsup. So you will receive the actual message send to participants.

**Comment by jlguardi** [comment]
The best way to identify the issue is add a new participant to the group using yowsup. So you will receive the actual message send to participants.

**Comment by ffwraspi** [comment]
thanks a lot for all your suggestions and your suport.
Group message is working now.

**Comment by jlguardi** [comment]
@ffwraspi Please, post which was the final solution and close this issue.

**Comment by ffwraspi** [comment]
The solution is/was:

yowsup was on the latest version

sowerkoku was right:
1. I saved the phone number from the rasperry pi into my contacts
2. I sent a message from the pi to the phone
3. I sent a message from the phone to the pi
4. I created a group
5. I sent a message from the phone into the group
6. I sent a message from the pi into the group 
=> group message receives all clients

I think it's imoprtant to write single messages between the clients/pi befor you create a group
